shell/platform/darwin/ios/framework/Source/FlutterEngineTest_mrc.mm

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+
+#import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine_Internal.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine_Test.h"
+#import "flutter/shell/platform/darwin/ios/platform_view_ios.h"
+
+FLUTTER_ASSERT_NOT_ARC
+
+@interface FlutterEngineTest_mrc : XCTestCase
+@end
+
+@implementation FlutterEngineTest_mrc
+
+- (void)setUp {
+}
+
+- (void)tearDown {
+}
+
+- (void)testSpawnsShareGpuContext {
```

Comment:
Because of our smart pointers around objective-c objects in headers.  Certain internal headers for the engine can't be included without blowing up.  In particular `FlutterEngine_Internal.h`.  I tried to fix our smart pointers but it's a losing battle.